packages/core/src/App/Containers/RealAccountSignup/real-account-signup.jsx

```diff
@@ -1,9 +1,10 @@
 import classNames             from 'classnames';
 import {
+    Button,
     Modal,
     Loading }                 from 'deriv-components';
-import React, { Component }   from 'react';
 import { localize, Localize } from 'deriv-translations';
+import React, { Component }   from 'react';
```

Comment:
>`react` import should occur before import of `deriv-translations`  import/order

travis is failing with this error

Reply:
Why do you think linter is throwing? shouldn't `d` come before `r`?